Gemfile

```
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "rails", "6.1.5"
+gem "rails", "7.0.2.3"
```

Would it be worth making this a `~>` definition at this point?

```suggestion
gem "rails", "~> 7.0.2.3"
```

---

Unfortunately not, as bundler may downgrade Rails if a dependency requires an older version.  It seems fixing the release is the safest option.